tools/generate_manifest_schema.py

```diff
@@ -0,0 +1,37 @@
+import json
```

Comment:
Can we add some documentation inside this script or as a `README.md` file in this folder?

Reply:
I updated the script to add some comments. `README.md` does not contain a lot of details, so I prefer to add the comments to the script itself.

Reply:
idea for future improvement: we can add a manual GH Action to run this script and raise a PR. The GHA will show how this script is used, and can save a bit of effort for running the python script manually.